core/src/main/resources/db/migration/V27__add_auto_install.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE `AutoInstall` (
```

Comment:
We want to automatically install whenever a version of this package is published (e.g every push to treehub) so I don't have a version number and therefore not a package_uuid.